src/LearningObjects/LearningObjectInteractor.ts

```diff
@@ -1164,21 +1164,11 @@ async function loadChildObjectSummaries({
     full: false,
     status: childrenStatus,
   });
-  children = await Promise.all(
-    // FIXME: Children should be mapped to LearningObjectSummary type which doesn't include materials or files
-    children.map(async child => {
-      child.materials = await dataStore.getLearningObjectMaterials({
-        id: child.id,
-      });
-      child.materials.files = await Gateways.fileMetadata().getAllFileMetadata({
-        requester,
-        learningObjectId: child.id,
-        filter: 'unreleased',
-      });
-      return child;
-    }),
-  );
-  return children;
+  let childSummary = children.map(child => {
```

Comment:
We can simplify this to
```suggestion
  let childSummary = children.map(mapLearningObjectToSummary)
```